tests/core/test_requirements_scanner.py

```diff
@@ -1,8 +1,26 @@
+import pkg_resources
+
 from guarddog.scanners.project_scanner import RequirementsScanner
 
 
 # Regression test for https://github.com/DataDog/guarddog/issues/78
 def test_requirements_scanner_on_non_existing_package():
     scanner = RequirementsScanner()
-    result = scanner.parse_requirements(["not-a-real-package==1.0.0"])
+    result = scanner.parse_requirements(["not-a-real-package==1.0.0", "flask==2.2.2"])
     assert "not-a-real-package" not in result
+    assert "flask" in result
```

Comment:
This assertion belong to a different test (or we can just rename this one to `test_requirements_scanner` to encompass both checks)